modules/ROOT/pages/api-proxy-landing-page.adoc

```diff
@@ -33,7 +33,50 @@ API Proxy version 2.0.0 and later add native support for OAS API specification a
 
 You can specify the version of the proxy you want to use by clicking the *Advanced Options* drop-down menu and selecting the proxy version that best covers your needs.
 
-See xref:proxy-deploy-raml-oas-proxy.adoc[Building a RAML/OAS API Proxy] for more information.
+== API Proxy Validations
+
+With Mule 4, you can perform strict validations directly from API Manager. Strict validations enable the application to compare a request with proxy specifications and allow only those headers and query parameters that are specified in the API definition. 
+
+You can perform strict validations for only query parameters, only headers, or for both. This feature is currently available only for RAML and WSDL proxies in Mule 4.
+
+If you do not enable strict validations, the parameters are still validated against the API definition. However, if the proxy receives a query parameter or a header that is not specified in the definition, that parameter is irrespectively sent to the backend service.
+
+You can  modify validation settings in API Manager when you create a proxy and deploy it using Runtime. You must use a proxy version later than 2.1.0 when modifying or creating a proxy. 
+
+
+=== API Specifications for Strict Validations
+
+When enabling strict validations for Mule 4 Applications, you must consider its architecture before designing the API specification. If your Mule application implements a proxy or a load balancer in front, the web server might modify the client request to include additional details, such as headers. In such cases, the Mule Application denies the request due to a non-valid header error. 
```

Comment:
I would change "you must consider its architecture"  with "you must consider the architecture you are including it" or something similar. We try to warn the users to consider the layers, such as proxies or load balancers around the mule runtime / or application, before enabling strict headers.

Reply:
Done! Thanks!